mods/d2k/weapons.yaml

```diff
@@ -83,68 +106,92 @@ M_LMG:
 		ContrailDelay: 0
 		ContrailUsePlayerColor: true
 	Warhead@1Dam: SpreadDamage
-		Spread: 96
-		Damage: 30
-		InvalidTargets: Structure
+		Spread: 48
+		Falloff: 100, 100, 65, 30, 0
+		Damage: 125
+		ValidTargets: Ground
 		Versus:
-			Light: 60
-			Heavy: 10
+			none: 100
+			wall: 10
+			building: 25
+			wood: 75
+			light: 40
+			heavy: 20
+			concrete: 100
+			invulnerable: 0
+			cy: 20
+			harvester: 25
 		DamageTypes: Prone50Percent, TriggerProne, BulletDeath
 	Warhead@2Eff: CreateEffect
 		Explosion: piffs
 
 M_HMG:
-	ReloadDelay: 10
-	Range: 5c0
-	MinRange: 1c0
+	ReloadDelay: 40
+	Range: 3c512
 	Report: 20MMGUN1.WAV
 	Projectile: Bullet
 		Speed: 1c256
-		Blockable: false
+		ContrailLength: 3
+		TrailInterval: 1
+		ContrailDelay: 0
+		ContrailUsePlayerColor: true
 	Warhead@1Dam: SpreadDamage
-		Spread: 96
-		Damage: 20
+		Spread: 48
+		Falloff: 100, 100, 65, 30, 0
+		Damage: 250
 		Versus:
-			Wood: 50
-			Light: 60
-			Heavy: 25
-			Concrete: 25
+			none: 25
+			wall: 100
+			building: 50
+			wood: 65
+			light: 100
+			heavy: 50
+			concrete: 100
+			invulnerable: 0
+			cy: 20
+			harvester: 50
 		DamageTypes: Prone50Percent, TriggerProne, BulletDeath
 	Warhead@2Eff: CreateEffect
 		Explosion: piffs
 
 Fremen_L:
-	ReloadDelay: 60
+	ReloadDelay: 40
 	Delay: 5
-	Range: 5c512
+	Range: 3c512
 	Report: BAZOOK2.WAV
-	InvalidTargets: Infantry
+	ValidTargets: Ground
 	Projectile: Bullet
-		Speed: 320
-		Blockable: false
-		Shadow: yes
-		Angle: 88
-		Inaccuracy: 384
-		Image: MISSILE
+		Speed: 1c256
+		ContrailLength: 3
+		TrailInterval: 1
+		ContrailDelay: 0
+		ContrailUsePlayerColor: true
 	Warhead@1Dam: SpreadDamage
-		Spread: 192
-		Damage: 30
-		InvalidTargets: Infantry
+		Spread: 64
+		Falloff: 100, 100, 65, 30, 0
+		Damage: 250
+		ValidTargets: Ground
 		Versus:
-			Wood: 75
-			Light: 40
-			Heavy: 90
-			Concrete: 50
+			none: 25
+			wall: 100
+			building: 50
+			wood: 65
+			light: 100
+			heavy: 50
+			concrete: 100
+			invulnerable: 0
+			cy: 20
+			harvester: 50
 		DamageTypes: Prone50Percent, TriggerProne, SmallExplosionDeath
 	Warhead@2Eff: CreateEffect
 		Explosion: small_explosion
-		ImpactSound: EXPLLG5.WAV
+		ImpactSound: EXPLSML1.WAV
 
 HMG:
-	ReloadDelay: 30
-	Range: 5c0
-	Burst: 2
-	BurstDelay: 5
+	ReloadDelay: 20
+	Range: 3c0
+#	Burst: 3	# Actually fires 3 times in D2k but deals damage all at once, purely visual/audio effect
```

Comment:
That wouldn't reproduce the original behavior either.

I opted fro `Burst: 3` and `BurstDelay: 2` with `Damage: 60` per burst instead. That looks and sounds pretty much like the original.